pkgs/development/compilers/elm/default.nix

```diff
@@ -247,11 +247,12 @@ in lib.makeScope pkgs.newScope (self: with self; {
           # see upstream issue https://github.com/dillonkearns/elm-pages/issues/305 for dealing with the read-only problem
           preFixup = ''
             patch $out/lib/node_modules/elm-pages/generator/src/codegen.js ${./packages/elm-pages-fix-read-only.patch}
+            patch $out/lib/node_modules/elm-pages/generator/src/init.js ${./packages/elm-pages-fix-init-read-only.patch}
           '';
 
           postFixup = ''
             wrapProgram $out/bin/elm-pages --prefix PATH : ${
-              with pkgs.elmPackages; lib.makeBinPath [ elm elm-review elm-optimize-level-2 ]
+              with pkgs.elmPackages; lib.makeBinPath [ elm elm-review elm-optimize-level-2 lamdera ]
```

Comment:
I'm considering reverting this one, since we're "prefixing" PATH, the user can add lamdera themselves. (context: lamdera is UNFREE).

Reply:
I think we should not include unfree packages by default. Nothing against the unfree software or lamdera specifically but should be user opt-in. It's expected that by default nixpkgs won't attempt to add unfree software.

That's being said I think this package should be BSD-3-Clause right? Just the service itself is unfree.